radixdlt/src/integration/java/com/radixdlt/consensus/simulation/SimulatedBFTNetwork.java

```diff
@@ -176,8 +180,24 @@ public SystemCounters getCounters(ECKeyPair keyPair) {
 		return counters.get(keyPair);
 	}
 
-	public void start() {
+	public Completable start() {
+		// Send start event once all nodes have reached real epoch event (first epoch event is empty)
```

Comment:
Why is the first epoch event empty again? And is it possible (now or ever) that more than just the first event is empty?

Reply:
its just to signify that there is no validator set in the beginning of a nodes lifetime.